tensorflow/core/kernels/matmul_op_impl.h

```diff
@@ -725,8 +725,12 @@ struct LaunchBatchMatMul<GPUDevice, Scalar> {
 
         BlasScratchAllocator scratch_allocator(context, max_scratch_size);
         auto blas = stream->parent()->AsBlas();
+        
         OP_REQUIRES(context, blas != nullptr,
                     absl::InternalError("No blas support for stream"));
+        
+        // TODO: calling hipblaslt here to replace blas->DoBlasGemmBatched        
```

Comment:
I guess this is copied from https://github.com/ROCm/tensorflow-upstream/commit/fbef0b5a170e204c06eb035911f043d7abbfd5f7 we can remove this now.